assets/src/data/model/base-date-formatter.js

```diff
@@ -105,7 +105,7 @@ export const datesToMysqlFormatOnEntities = (
  *   to the local timezone for the host.
  * @return { Object }  Returns a new entity with the date field values formatted
  */
-export const datesToMysqlFormatOnEntity = (
+export const formatEntityDateToMysql = (
```

Comment:
sry for being nitpicky.. but shouldn't it be `formatEntityDatesToMysql()` because it may format more than one date?

Reply:
ya good catch